lib/src/test/java/org/team100/lib/localization/TestCameraAngles.java

```diff
@@ -53,4 +53,30 @@ void testCameraAngles4() {
             assertEquals(1, testTranslation1.getY(), 0.0001);
         }
     }
+
+    @Test
+    void testCameraAngles5() {
+        {
+            Transform3d e = new Transform3d(0, 0, 1, new Rotation3d(0, Math.toRadians(45), Math.PI));
```

Comment:
add comments for each line in these tests explaining what geometry you are verifying